package.json

```diff
@@ -49,38 +49,39 @@
   },
   "homepage": "https://github.com/yusinto/universal-hot-reload#readme",
   "dependencies": {
-    "@types/webpack": "^4.32.1",
-    "babel-loader": "^8.0.6",
+    "@types/webpack": "4.41.21",
+    "babel-loader": "8.1.0",
+    "clear-require": "^3.0.0",
```

Comment:
Does this need to omit the ^ to be consistent with the other package versions?

Reply:
No, this should be fine as we'll always want the latest version. I think it's fine.